content/ru/docs/intro/using_helm.md

```diff
@@ -0,0 +1,482 @@
+  scenarios where Deployment has `replicas` set to 1 and `maxUnavailable` is not
+  set to 0 as part of rolling update strategy, `--wait` will return as ready as
+  it has satisfied the minimum Pod in ready condition.
+- `--no-hooks`: Это пропускает запуск hooks для команды
+- `--recreate-pods` (доступно только для `upgrade` и `rollback`): Этот флаг
+  приведет к пересозданию всех pods (with the exception of pods belonging to
+  deployments). (УСТАРЕЛО в Helm 3)
+
+## 'helm uninstall': Удаление Релиза
+
+Когда потребуется удалить релиз из кластера, используйте команду `helm uninstall`:
+
+```console
+$ helm uninstall happy-panda
+```
+
+Это приведет к удалению релиза из кластера. Вы можете просмотреть все ваши текущие
+развернутые релизы с помощью команды `helm list` :
+
+```console
+$ helm list
+NAME            VERSION UPDATED                         STATUS          CHART
+inky-cat        1       Wed Sep 28 12:59:46 2016        DEPLOYED        alpine-0.1.0
+```
+
+Из примера выше видно, что релиз `happy-panda` был удален.
+
+В предыдущих версиях Helm, когда релиз удалялся, запись о его
+удалении оставалась. В Helm 3, Удаление удаляет записи релиза, а также.
+Если вы хотите сохранить запись об удалении выпуска, используйте `helm uninstall
+--keep-history`. Использование `helm list --uninstalled` покажет только те выпуски, которые
+были удалены с флагом `--keep-history`.
+
+Флаг `helm list --all` покажет вам все релизы,
+сохраненные Helm, включая записи для неудачных или удаленных элементов (если был указан параметр `--keep-history`
+):
+
+```console
+$  helm list --all
+NAME            VERSION UPDATED                         STATUS          CHART
+happy-panda     2       Wed Sep 28 12:47:54 2016        UNINSTALLED     wordpress-10.4.5.6.0
+inky-cat        1       Wed Sep 28 12:59:46 2016        DEPLOYED        alpine-0.1.0
+kindred-angelf  2       Tue Sep 27 16:16:10 2016        UNINSTALLED     alpine-0.1.0
+```
+
+Обратите внимание, что поскольку релизы теперь удаляются по умолчанию, откат удаленного ресурса больше невозможен.
+
+## 'helm repo': Работа с Репозиториями
+
+Helm 3 больше не поставляется со стандартным хранилищем chart-ов. Команды `helm repo`
+предоставляет возможности добавления, перечисления и удаления репозиториев.
+
+Узнать о настроенных репозитории можно с помощью `helm repo list`:
+
+```console
+$ helm repo list
+NAME            URL
+stable          https://charts.helm.sh/stable
+mumoshu         https://mumoshu.github.io/charts
+```
+
+Новые репозитории могут быть добавлены с помощью `helm repo add`:
+
+```console
+$ helm repo add dev https://example.com/dev-charts
+```
+
+Поскольку репозитории диаграмм часто меняются, в любой момент вы можете убедиться,
+что ваш клиент Helm обновлен, запустив `helm repo update`.
+
+Репозитории могут быть удалены с помощью `helm repo remove`.
+
+## Создание Собственных Chart-ов
+
+[Руководство по разработке Chart-ов](https://helm.sh/docs/topics/charts/) объясняет, как
```

Comment:
How can I do"ref" without a direct link? The page with the Russian translation is not yet available